spec-0007/index.md

```diff
@@ -5,69 +5,60 @@ author:
   - "Stéfan van der Walt <stefanv@berkeley.edu>"
   - "Sebastian Berg <sebastianb@nvidia.com>"
   - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
   - Other participants in the discussion <not.yet@named.org>"
 discussion: https://github.com/scipy/scipy/issues/14322
 endorsed-by:
 ---
 
 ## Description
 
-<!--
-Briefly and clearly describe the proposal.
-Explain the general need and the advantages of this specific proposal.
-If relevant, include examples of how the new functionality would be used,
-intended use-cases, and pseudo-code illustrating its use.
--->
-
-There is disparity in the APIs libraries provide to seed random number generation.
-This SPEC suggests a single, pragmatic API for the ecosystem, taking into account technical and historical factors.
+Currently, libraries across the ecosystem provide various APIs for seeding random number generation.
+This SPEC suggests a unified, pragmatic API, taking into account technical and historical factors.
 Adopting such a uniform API will simplify the user experience, especially for those who rely on multiple projects.
 
-We recommend to:
+We recommend:
 
-- Standardize the usage and interpretation of an `rng` keyword for seeding, and
-- avoid the use of global state and legacy bitstream generators.
+- avoiding the use of global state and legacy bitstream generators, and
+- standardizing the usage and interpretation of an `rng` keyword for seeding.
 
-We suggest doing this by:
+We suggest implementing these principles by:
 
-- Passing the `rng` argument to `numpy.random.default_rng` to instantiate a `Generator`, and
-- deprecating the use of `np.seed` and `RandomState`.
+- deprecating the use of `RandomState` and `numpy.random.seed`,
+- accepting an `rng` argument in all functions that require random number generation, and
+- using `numpy.random.default_rng` to validate the `rng` argument and instantiate a `Generator`.
 
 ### Concepts
 
-- `BitGenerator`: Generates a stream of pseudo-random bits. The default generator in NumPy (`np.random.default_rng`) uses PCG64.
+- `BitGenerator`: Generates a stream of pseudo-random bits. The default generator in NumPy (`numpy.random.default_rng`) uses PCG64.
```

Comment:
Not sure if this is required or if the link would work either way? If it would work either way, I will revert.